.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,4 +18,4 @@
 
```

Comment:
How come this file is among the files changed in this branch/pull request? Did you add an additional line?

Reply:
Each time I created a new branch, I had to add or remove new line in other to have changes that would enable me to commit the file into my new branch